src/Auth/Process/OpaqueSmartID.php

```diff
@@ -441,19 +441,19 @@ private function parseUserId($attribute): string
     /**
      * @param   array  $idpMetadata
      *
-     * @return  string  IdPs list of emails
+     * @return  array  IdPs list of emails
      */
     private function getIdPEmailAddress(array $idpMetadata): string
     {
-        $idpEmailAddress = null;
+        $idpEmailAddress = [];
         if (!empty($idpMetadata['contacts']) && is_array($idpMetadata['contacts'])) {
             foreach ($idpMetadata['contacts'] as $contact) {
                 if (!empty($contact['contactType']) && !empty($contact['emailAddress'])) {
                     if ($contact['contactType'] === 'technical') {
-                        $idpEmailAddress = $contact['emailAddress'];
+                        $idpEmailAddress[] = $contact['emailAddress'];
                         continue;
                     } elseif ($contact['contactType'] === 'support') {
-                        $idpEmailAddress = $contact['emailAddress'];
+                        $idpEmailAddress[] = $contact['emailAddress'];
```

Comment:
Instead of changing lines 453 and 456 we can add the following after L463:
```
if (is_string($idpEmailAddress)) {
    // Convert $idpEmailAddress to an array
    $idpEmailAddress = explode(";", $idpEmailAddress);
}
```

In the end:
```
if (!empty($idpEmailAddress)) {
    if (is_string($idpEmailAddress)) {
        // Convert $idpEmailAddress to an array
        $idpEmailAddress = explode(";", $idpEmailAddress);
    }
    foreach ($idpEmailAddress as &$idpEmailAddressEntry) {
        ...
```